File: Examples/DotNET/CSharp/Document/MergePresentations.cs
using System;
using Com.Aspose.Slides.Api;
using Com.Aspose.Slides.Model;
using Com.Aspose.Storage.Api;

namespace Document
{
    class MergePresentations
    {
        public static void Run()
        {
            // ExStart:1
            SlidesApi slidesApi = new SlidesApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "sample.pptx";
            String storage = "";
            String folder = "";

            String mergeFile1 = "sample-input.pptx";
            String mergeFile2 = "demo.pptx";

            PresentationsMergeRequest body = new PresentationsMergeRequest();
            body.PresentationPaths = new System.Collections.Generic.List<string> { mergeFile1, mergeFile2 };

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));
                storageApi.PutCreate(mergeFile1, null, null, System.IO.File.ReadAllBytes(Common.GetDataDir() + mergeFile1));
                storageApi.PutCreate(mergeFile2, null, null, System.IO.File.ReadAllBytes(Common.GetDataDir() + mergeFile2));

                // Invoke Aspose.Slides Cloud SDK to merge PowerPoint presentations
                DocumentResponse apiResponse = slidesApi.PostPresentationMerge(fileName, storage, folder, body);

                if (apiResponse != null)
                {
                    // Download merged document from storage
                    Com.Aspose.Storage.Model.ResponseMessage storageRes = storageApi.GetDownload(fileName, null, null);
                    System.Diagnostics.Debug.WriteLine("response:" + storageRes.ResponseStream);
                    System.IO.File.WriteAllBytes(Common.GetDataDir() + fileName, storageRes.ResponseStream);
                    Console.WriteLine("Merge PowerPoint Presentations, Done!");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);

            }
            // ExEnd:1
        }
    }
}

File: Examples/Node.js/Document/MergePresentations.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var SlidesApi = require('asposeslidescloud');
var StorageApi = require('asposestoragecloud');
var configProps = require('../Config/config.json');
var data_path = '../../../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var outFolder = configProps.out_folder;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);
// Instantiate Aspose.Slides API SDK
var slidesApi = new SlidesApi(config);

// Set input file name
var name = "sample.pptx";
var mergeFile1 = "welcome.pptx";
var mergeFile2 = "demo.pptx";
var orderedMergeRequestBody = {
		'PresentationPaths' : [mergeFile1, mergeFile2]
};

try {

// Upload files to aspose cloud storage
storageApi.PutCreate(name, null, null, data_path + name , function(responseMessage) {
	assert.equal(responseMessage.status, 'OK');

	storageApi.PutCreate(mergeFile1, null, null, data_path + mergeFile1 , function(responseMessage) {
		assert.equal(responseMessage.status, 'OK');

		storageApi.PutCreate(mergeFile2, null, null, data_path + mergeFile2 , function(responseMessage) {
			assert.equal(responseMessage.status, 'OK');

			// Invoke Aspose.Slides Cloud SDK API to merge multiple presentation files
			slidesApi.PostPresentationMerge(name, null, null, orderedMergeRequestBody, function(responseMessage) {
					assert.equal(responseMessage.status, 'OK');

					// Download presentation from cloud storage
					storageApi.GetDownload(name, null, null, function(responseMessage) {
						assert.equal(responseMessage.status, 'OK');
						var writeStream = fs.createWriteStream(data_path + name);
						writeStream.write(responseMessage.body);
						});
				});
			});
		});
	});

}catch (e) {
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1